packages/client/tests/functional/issues/18292-inspect-loop/test.ts

```diff
@@ -0,0 +1,35 @@
+import { inspect } from 'util'
+
+import testMatrix from './_matrix'
+// @ts-ignore
+import type { PrismaClient } from './node_modules/@prisma/client'
+
+declare let prisma: PrismaClient
+
+testMatrix.setupTestSuite(() => {
+  beforeAll(async () => {
+    await prisma.user.create({ data: { email: 'user@example.com' } })
+  })
+
+  test('it is possible to inspect/log prisma client', () => {
+    expect(() => inspect(prisma)).not.toThrow()
+  })
+
+  test('result extensions are still logged/inspected correctly', async () => {
+    const xprisma = prisma.$extends({
+      result: {
+        user: {
+          computedField: {
+            needs: [],
+            compute() {
+              return 'HELLO!!!'
+            },
+          },
+        },
+      },
+    })
+
+    const user = await xprisma.user.findFirstOrThrow({ select: { computedField: true, email: true } })
+    expect(inspect(user)).toMatchInlineSnapshot(`{ email: 'user@example.com', computedField: 'HELLO!!!' }`)
```

Comment:
Should we add a test to include inspection options are respected (eg. depth)?

Reply:
Added